js/constants/constants.js

```diff
@@ -106,7 +106,10 @@ module.exports = {
     AUTO_QUALITY: "Auto",
     SHARE: "Share",
     MORE_OPTIONS: "More Options",
-    SELECT_TO_UNMUTE: "SELECT TO UNMUTE"
+    SELECT_TO_UNMUTE: "SELECT TO UNMUTE",
+    AUDIO: "Audio",
+    SUBTITLES: "Subtitles",
+    UNDEFINED_LANGUAGE: "Undefined language"
```

Comment:
Have we checked with Steve Paddon and Fernando I. about this string?

Reply:
Yeah, there's a [question](https://wiki.corp.ooyala.com/pages/viewpage.action?spaceKey=ENG&title=Supporting+Audio+Service+Type+in+OTS#link-talk-3468) in wiki on the topic.